ballerina-tests/constraint-validation-tests/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Ballerina generates this directory during the compilation of a package. 
```

Comment:
I think we don't need this file. Having a single .gitignore file in the root directory would be enough

Reply:
Updated.